recipes/lbcharmdb/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "lbcharmdb" %}
+{% set version = "0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lbcharmdb-{{ version }}.tar.gz
+  sha256: 99b330c91dded3b6eb21fbcba1fa49ad419c1158665a2e9f77daebe56418c1ff
+
+build:
+  entry_points:
+    - lbcharmdb-compute-correlation = lbcharmdb.calculate_correlations_weighted_tuples:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=61.2
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
+    - tqdm
+    - root_base
+    - numpy
+    - coloredlogs
+
+test:
+  imports:
+    - lbcharmdb
+  commands:
+    - pip check
+    - lbcharmdb-compute-correlation --help
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.cern.ch/lhcb-charm/correlations-database-editor
+  summary: Maintenance tools for the database of correlations between charm analyses
+  dev_url: https://gitlab.cern.ch/lhcb-charm/correlations-database-editor
+  license: GPL-2.0-or-later
```

https://gitlab.cern.ch/lhcb-charm/correlations-database-editor/-/blob/master/LICENSE?ref_type=heads

```suggestion
  license: GPL-3.0-only
```